.ci/packaging.groovy

```diff
@@ -394,8 +394,10 @@ def prepareE2ETestForPackage(String beat){
 
 def release(type){
   withBeatsEnv(type){
+    // As agreed DEV=false for staging otherwise DEV=true
+    // this should avoid releasing binaries with the debug symbols and disabled most build optimizations.
     withEnv([
-      "DEV=true"
+      "DEV=${!type.equals('staging')}"
```

Comment:
Which specific builds are staging? Release branches?

Reply:
the `release` function is called when generating the binaries, if a branch then it uses `staging` otherwise `snapshot`.

See https://github.com/elastic/beats/blob/e8df5e92dcb39beb909781d07c9aba7f602d9238/.ci/packaging.groovy#L289-L307